packages/pirania-app/files/usr/libexec/rpcd/pirania-app

```diff
@@ -8,11 +8,11 @@ You may obtain a copy of the License at
 ]]--
 
 require "ubus"
```

Comment:
could this file be called pirania instead of pirania-app?
in that way it would be pirania ubus interface, and it would make sense for others to use it for their own interfaces.

Reply:
There are two different apis. `pirania` deals with all the logic of the `voucher` system, while `pirania-app` deals more specifically with stuff related to the front-end app, such as content. Should it be merged into `pirania`?

Reply:
I feel that the '-app' doesn't represent what it is actually doing.
Adding a piece of documentation to it would help figure out why they are grouped the way they are... and may be merging or renaming them. Thanks!

Reply:
Docs have been added. Lemme know what u think. This decision was made by @altergui and me when we first got pirania working, and I built the vanilla JS app. The distinction between pirania and pirania-app may not make much sense now anymore.

Reply:
it might not be needed anymore indeed

Reply:
thanks for the docs!
It might be good to describe the whole governance.json.
it seems it has some clear structure with specific values that are used in different parts... would be good to know how them affect the use of the app.

Reply:
thanks for the docs!
It might be good to describe the whole governance.json.
it seems it has some clear structure with specific values that are used in different parts... would be good to know how them affect the use of the app.